### ISSUE TITLE
API is now out of date.

### ISSUE THREAD

**Issue by minthat** [TheZ3ro/pkmngo-map#6]
Check out the new API here:

https://github.com/tejado/pgoapi

**Comment by TheZ3ro** [comment]
I've seen that before, but how about Fort scanning and pokemon scanning?

**Comment by dassaffe** [comment]
Well that was a short lifetime. So, it isn't just a misconfiguration on my side, the api is just down and we can't create maps / display pokemon right now? Just so I understand what is going on.

**Comment by minthat** [comment]
No, P-GO servers are down right now...

**Comment by NumberXer0** [comment]
The servers are up and the map still doesn't populate. Localhost:5000 does not load, but the scans work fine. Just no visual map.

**Comment by minthat** [comment]
Default port is 8000. Servers are extremely unstable atm.

**Comment by NumberXer0** [comment]
Whoops, typo. I meant 8000. But either way, it says Google Maps wasn't loaded correctly. I guess it is the servers.

**Comment by minthat** [comment]
Google Maps wasn't loaded correctly? The error you should be getting is an api session error. Once everything back online post a new issue with your error logs

**Comment by NumberXer0** [comment]
Yeah, the page says it isn't loaded correctly and the console says Google Maps API Error: ApiNotActivatedMapError. Not sure if it's related to the API being outdated or something else though.

**Comment by minthat** [comment]
Lol glad you figured it out. It was a bad gmaps api key right?
